### ISSUE TITLE
Make it easier to customise the typography

### ISSUE THREAD

**Issue by paulrobertlloyd** [frctl/mandelbrot#42]
Currently, Open Sans is applied liberally throughout the theme (by means of using the `font` shorthand property), whereas it only need be applied once to the `html` element, and then inherited by those child elements that share that type styling (which probably means stating the `font-size` and `line-height` properties separately within the typographic mixins).

By stating this font choice only once, it would then be easier to change this setting in any subsequent theme CSS overrides.

**Comment by paulrobertlloyd** [comment]
Note for @allmarkedup: Happy to fix this; should be a case of updating some of the SCSS mixins. I’m tempted to add this to my currently outstanding PR, but I know that’s bad practice. Yet weary of opening another PR, especially when there are a number of different versions of mandlebrot floating about already. Hmmm… open to suggestions on how to approach.

**Comment by allmarkedup** [comment]
Maybe hold off on this for now mate - I've been trying to figure out a way to improve customisation of stuff like this but I wouldn't mind having a chat with you about it at some point to get a different viewpoint on it all. Any chance I can tempt you into the office sometime soon(ish)?
